daemon/cancel_or_refund_test.go

```diff
@@ -61,13 +63,13 @@ func TestXMRTakerCancelOrRefundAfterKeyExchange(t *testing.T) {
 	go func() {
 		defer wg.Done()
 		for {
-			count, err := ec.PendingTransactionCount(ctx) //nolint:govet
+			aliceNonce, err := ec.PendingNonceAt(ctx, aliceAddr) //nolint:govet
```

Comment:
Another test from another package can be running and make us think the `newSwap` contract method was called when using `PendingTransactionCount`, but since Alice's ganache key is not shared between packages, an incremented nonce check is safe.

I looked at other options before this simple one. Without a transaction ID, I don't see any API to pull full pending transaction data from the mempool. I tried creating a separate client to the DB, which may not even be safe (don't know enough about the underlying DB), but the 2nd client wasn't seeing the new TX ID as soon as Alice's client added the entry ... there was significant delay. I could be wrong, but I don't think you can use filters with pending transactions.